fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java

```diff
@@ -1812,6 +1844,106 @@ public TStreamLoadMultiTablePutResult streamLoadMultiTablePut(TStreamLoadPutRequ
         return result;
     }
 
+    public class ReportStreamLoadWorker implements Runnable {
```

Comment:
Add comment to explain what is this class for?
And is it suitable to make it as a inner class of `FrontendServiceImpl`?
It will make `FrontendServiceImpl` too large

Reply:
Remove this first, it shouldn't be necessary